docs/install-on-microk8s.md

```diff
@@ -0,0 +1,51 @@
+# Install on MicroK8s
+
+## Pre-requisites
+- [MicroK8s](https://microk8s.io/)
+- To install etcd using these instructions, you must have [Helm](https://helm.sh) in your client environment.
+- [Download the Compose on Kubernetes installer](https://github.com/docker/compose-on-kubernetes/releases).
+- Ensure storage and dns are enabled: `microk8s.enable storage dns`
+- `kubectl` availabe in path - if it's not already available it can be aliased from the snap with: `sudo snap alias microk8s.kubectl kubectl`.
+
+## Create compose namespace
+
+- Confirm that MicroK8s is working with `microk8s.inspect`.
+- Create a compose namespace by running `kubectl create namespace compose`.

```

Comment:
as the ~/.kube/config file has not yet been generated, prefer microk8s.kubectl:
```suggestion
- Create a compose namespace by running `microk8s.kubectl create namespace compose`.
```

Reply:
Or would it be better to move this section beneath `Populate Kubernetes Config` below?

Reply:
Fair enough fixed in 9157c6a.